src/main/java/io/jenkins/plugins/restlistparam/RestListParameterValue.java

```diff
@@ -20,19 +20,26 @@
   @Restricted(NoExternalUse.class)
   public String value;
 
+  @Exported(visibility = 4)
+  @Restricted(NoExternalUse.class)
+  public String displayValue;
```

Comment:
Why do you need to persist this value in this Class? you are not using the display value within the pipeline and for `Replay` (with the replay plugin) you don't need that either since that will re-query the API anyway?